cmd/argocd/commands/tree.go

```diff
@@ -0,0 +1,75 @@
+package commands
```

Comment:
Please add unit tests for all functionality in this file as this is the main code for the tree view

Reply:
sure Leo

Reply:
sure Leo.